websites/R/RadiosNet/metadata.json

```diff
@@ -0,0 +1,23 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.9",
+	"author": {
+		"id": "784546644766621746",
+		"name": "ntn_ss"
+	},
+	"service": "RadiosNet",
+	"description": {
+		"en": "The most famous radio aggregator on the Brazilian Internet, active since 1997.",
+		"pt_BR": "O agregador de rádios mais famoso da internet brasileira, em atividade desde 1997."
+	},
+	"url": "radios.com.br",
+	"version": "1.0.0",
```

Comment:
There is a new field called `matches`, which is required for new Presences.
These are an array of strings which need to follow https://developer.chrome.com/docs/extensions/develop/concepts/match-patterns.